3-UsingAPIs/Week2/homework/ex6-browserDebug/index.js

```diff
@@ -39,12 +49,12 @@ function renderLaureates(laureates) {
 
 async function fetchAndRender() {
   try {
-    const laureates = getData(
+    const { laureates } = await getData(
```

Comment:
👍  Well done. I hope you learned about JS debugging - this is a very useful skill to learn